CHANGELOG.md

```diff
@@ -8,10 +8,19 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [2.3.0] - 2019-02-22
+
 ### Added
 
 - Replace shaka-player by dashjs.
 - Play timed text tracks in player.
+- Enable timex text tracks in dashboard.
+- Display transcripts along side video.
+
+### Improvements
+
+- Real languages display in the player (captions section).
+- Language choices are fetch once and then cached in the redux store.
```

Comment:
You must also modify the links down this file.

Reply:
Done